examples/example_plugin/example_plugin/template_content.py

```diff
@@ -1,6 +1,62 @@
+from django.urls import reverse
+
 from nautobot.extras.plugins import PluginTemplateExtension
 
 
+class CircuitContent(PluginTemplateExtension):
+    model = "circuits.circuit"
+
+    def detail_tabs(self):
+        """
+        You may define extra tabs to render on a model's detail page by utilizing this method.
+        Each tab is defined as a dict in a list of dicts.
+
+        For each of the tabs defined:
+        - The <title> key's value will become the tab link's title.
+        - The <url> key's value is used to render the HTML link for the tab.
+
+        These tabs will be visible (in this instance) on the Circuit model's detail page as
+        set by the CircuitContent.model attribute "circuits.circuit"
+
+        This example demonstrates defining one tab.
+        """
+        return [
+            {
+                "title": "Plugin Tab",
+                "url": reverse("plugins:example_plugin:circuit_detail_tab", args=[self.context["object"].pk]),
```

Comment:
I'd recommend using `kwargs` rather than `args` to reverse the URL more explicitly:

```suggestion
                "url": reverse("plugins:example_plugin:circuit_detail_tab", kwargs={"pk": self.context["object"].pk}),
```